rnd/autogpt_server/pyproject.toml

```diff
@@ -53,8 +53,8 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry.scripts]
 app = "autogpt_server.app:main"
 cli = "autogpt_server.cli:main"
-format = "code:format"
-lint = "code:lint"
+format = "some_code:format"
```

Comment:
Can we rename it anything other than some_code ?
code_quality.py, code_formatter.py, lint.py

Reply:
Happy to, just was in my stack for the night to merge. We could forgo this file altogether using Poe the poet (see the build scripts in the pyproject.toml)

Reply:
Renamed this to linter.py